decentralised-identity.md

```diff
@@ -0,0 +1,138 @@
+- Need-to-Know Basis: Only share what's necessary. If someone just needs to confirm your age, that's all they get. No unnecessary details get thrown into the mix, keeping your personal info on a need-to-know basis.
+- Reduced Data Collection: DID cuts down on the data hoarding. Traditional systems often scoop up more info than they need. With decentralized identity, the focus is on minimalism, collecting only what's essential for a particular interaction.
+- User Empowerment: You're in charge. You decide who gets what, giving you the power to manage your digital identity. This approach puts a strong emphasis on user empowerment, ensuring that you're not at the mercy of entities collecting more data than necessary.
+- Mitigating Surveillance Risks: By minimizing the information shared, DID acts as a shield against unnecessary surveillance. It aligns with the cypherpunk principles of resisting unwarranted intrusion into personal lives.
+
+In a nutshell, decentralized identity lets you share just enough to get things done while keeping the rest safely tucked away. It's privacy on your terms.
+
+Can privacy be achieved?
+
+Selective disclosure mechanisms allow users to share specific attributes without revealing their entire identity, reducing the risk of unnecessary exposure. While DID holds great potential for enhancing privacy, its effective implementation relies on careful design, robust encryption, and user education to prevent potential pitfalls or misuses of this technology.
+
+### Interoperability:
+
+Interoperability in decentralized identity is like making sure different digital systems speak the same language. It's about ensuring that your digital identity can smoothly work across various platforms and services without any hiccups.
+
+Imagine if your ID card only worked with one type of door lock. That would be inconvenient, right? Interoperability solves this issue. It means your decentralized identity can be recognized and used on different apps, websites, or services, just like your ID works wherever you go.
+
+Imagine buying things online without constantly entering your details. Using blockchain, your payment details become tamper-resistant and transparent. No single authority holds all your financial info, reducing the risk of a massive data breach. This isn't just about security; it's about convenience. With a decentralized identity, online payments become more streamlined, less prone to fraud, and entirely under your control. 
+
+The key is having common standards and rules that everyone follows. This way, whether you're logging into a social media account, accessing a financial service, or using an online marketplace, your decentralized identity seamlessly fits. It's like having a universal key for your digital interactions, making things more convenient and user-friendly. Interoperability is the glue that holds together the vision of a decentralized identity.
+
```

Comment:
Yes, it would be more interoperability using your DID on different systems. e.g. The ability to interact with and operate with different websites only sharing the bare minimum of what is required. Not sure should I change the heading to 'Compatibility' ? Like having a DID that is compatible with multiple systems?

Reply:
I have added the following piece and updated the section to Compatibility: However, if there's a lack of compatibility in decentralized identity, it's akin to digital systems speaking different languages, resulting in a fragmented and inconvenient user experience. In this scenario, your decentralized identity might struggle to smoothly navigate across various platforms and services, creating hiccups and complexities akin to having an ID card that only works with specific door locks.